app/views/users/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<div class='post-container'>
+    <div class='post-img'>
+        <img src='https://i.pinimg.com/originals/73/16/f5/7316f550de9ca0045e3d8d98a5bb5e44.png' alt='post' >
+    </div>
+    <div class='post-wrapper'>
+        <h1> <%= @user.name %> </h1>
+        <p> Number of posts: <%= @user.posts_counter %> </p>
+    </div>
+</div>
+
+<div class='bio-div'>
+    <h4> Bio</h4>
+    <p> <%= @user.bio %> </p>
+
+</div>
+
+<% @posts.each do |post| %>
+    <%= link_to user_posts_path(@user.id) do %>
+        <div class='post-div'>
+            <h4> <%= post.title %> </h4>
+            <p> lorem ipsum deprocadi ips deasdflorem ipsum deprocadi ips deasdflorem ipsum deprocadi ips deasdflorem ipsum deprocadi ips deasdflorem ipsumdeasdflorem ipsum deprocadi ips deasdflorem ipsum ... </p>
+        </div>
+    <% end %>
+<% end %>
+
+
+<div class='button'>
+    <button> See all Posts </button>
```

Comment:
- [ ] This button(**See all Post**) should link to user's post index as displayed on the [wireframe](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_user_all_posts.png?token=ABIFLA3SVPWT6MXQ3C2OGFLBHIHES). Please make it a link to all user's posts.